Automattic/sensei
# Update Composer dependencies

## Proposed Changes

* Update the Composer dependencies to be in line with PHP 7.3
* Update PHPUnit to 9.6
* Fix tests and deprecation warnings
* Fix CI

## Testing Instructions


* Delete the `vendor` and `third-party` folders. Run `composer install`.
* Everything should be working as before.
* CI should be passing.

## Pre-Merge Checklist
